scripts/limsquery.py

```diff
@@ -0,0 +1,24 @@
+
+from genologics.entities import *
+from genologics.lims import *
+from genologics.config import BASEURI, USERNAME, PASSWORD
+
+
+
+
+lims= Lims(BASEURI, USERNAME, PASSWORD)
+
+
+def  get_udfs():
+
+    step_name='End repair, A-tailing and adapter ligation (TruSeq RNA) 4.0'
```

Comment:
yeah!! I can see this is just a small utility script, but wouldn't it be more cool to take the `step` and `udf` name from command line or a file ? So this script will be untouched. But again, just asking :)